UnitySDK/Assets/ML-Agents/Scripts/Sensor/ISensor.cs

```diff
@@ -8,6 +10,79 @@ public enum SensorCompressionType
         PNG
     }
 
+    public class WriteAdapter
```

Comment:
should there just be two of these, one for `List<float>` and one for `TensorProxy`?

Reply:
I intended it to be non-abstract; trying to avoid a virtual call every time we need to write a float (although it's still branch-y in the worst case)